lib/WUI/wui.c

```diff
@@ -122,7 +122,11 @@ void StartWebServerTask(void const *argument) {
 }
 
 const char *wui_get_api_key() {
+#ifdef _DEBUG
+    return "PL-mini";
```

Comment:
I understand the motivation, but this seems problematic for two reasons:

* It is quite reasonable to expect developers or even some tinkerer users to run a development version of the firmware and use the printer the usual way, some kind of background testing. Having a well-known password that's impossible to change looks like a serious security issue, especially when evil GCode can lead to HW damage…
* This is surprising. If I read it correctly, when user clicks on the „Generate API Key“ to change it, it generates one, stores it into EEPROM, shows it, but will still use the old one.

If your motivation is to make it easier to repeatedly reflash & lose the eeprom content, I'd suggest we allow loading API key from the config file. When you reflash, you need to set the SSID/Password from there anyway. It'll also be a nice feature for people running multiple printers, so they can set the same API key on all of them if they like.

Should I create a jira ticket for that?

Reply:
perfect...I'm looking forward for the USB configuration usage